impeller/aiks/canvas.cc

```diff
@@ -300,4 +302,19 @@ void Canvas::DrawTextFrame(TextFrame text_frame, Point position, Paint paint) {
   GetCurrentPass().AddEntity(std::move(entity));
 }
 
+void Canvas::DrawVertices(Vertices vertices,
+                          Entity::BlendMode mode,
+                          Paint paint) {
+  std::shared_ptr<VerticesContents> contents =
+      std::make_shared<VerticesContents>(vertices);
+  contents->set_color(paint.color);
+  Entity entity;
+  entity.SetTransformation(GetCurrentTransformation());
+  entity.SetStencilDepth(GetStencilDepth());
+  entity.SetBlendMode(mode);
```

Comment:
This should be the paint blend mode.

The `drawVertices` blend parameter is used for blending the vertex colors (if they're available) with the paint color, which is separate from the blend operation that happens against the render backdrop.

Reply:
TIL thats what it does